contracts/contracts/interfaces/IVerifiersManager.sol

```diff
@@ -0,0 +1,50 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+interface IVerifiersManager {
+
+    error ZERO_ADDRESS();
+
+    struct RSAProveCircuitProof {
```

Comment:
change 'RSAProveCircuitProof' for 'ProveCircuitProof' as ecdsa signature is also used to sign passports